setup.py

```diff
@@ -18,12 +18,12 @@
     long_description = f.read()
 
 setup(
-    name='adafruit-circuitpython-drv2605',
+    name='adafruit-circuitpython-ds2413',
 
     use_scm_version=True,
     setup_requires=['setuptools_scm'],
 
-    description='CircuitPython library for DRV2605 haptic motor controller.',
+    description='CircuitPython library for DS2413 haptic motor controller.',
```

Comment:
this is a 'DS2413 1-Wire Dual Channel Addressable Switch'

Reply:
Oops